apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -261,7 +264,12 @@ handle_request('GetName', Req, _Context) ->
             {200, [], #{name      => Name,
                         name_hash => aec_base58c:encode(name, Hash),
                         name_ttl  => NameTTL,
-                        pointers  => Pointers}}
+                        pointers  => Pointers}};
+        {error, name_not_found} ->
+            {404, [], #{reason => <<"Name not found">>}};
+        {error, Reason} ->
+            ReasonBin = atom_to_binary(Reason, utf8),
```

Comment:
This is relying on undocumented/unspec-ed `aec_conductor:get_name_entry` return type.

Reply:
Spec changed to `atom()`.